tests/checks/mock/test_docker_daemon.py

```diff
@@ -0,0 +1,106 @@
+# stdlib
+import mock
+
+# project
+from tests.checks.common import AgentCheckTest
+
+MOCK_CONFIG = {
+    "init_config": {},
+    "instances": [{
+        "url": "unix://var/run/docker.sock",
+        "collect_disk_stats": True,
+    }]
+}
+
+
+class TestCheckDockerDaemon(AgentCheckTest):
+    CHECK_NAME = 'docker_daemon'
+
+    def mock_normal_get_info(self):
+        return {
+            'DriverStatus': [
+                ['Data Space Used', '1 GB'],
+                ['Data Space Available', '9 GB'],
+                ['Data Space Total', '10 GB'],
+                ['Metadata Space Used', '1 MB'],
+                ['Metadata Space Available', '9 MB'],
+                ['Metadata Space Total', '10 MB'],
+            ]
+        }
+
+    def mock_get_info_no_used(self):
+        return {
+            'DriverStatus': [
+                ['Data Space Available', '9 GB'],
+                ['Data Space Total', '10 GB'],
+                ['Metadata Space Available', '9 MB'],
+                ['Metadata Space Total', '10 MB'],
+            ]
+        }
```

Comment:
We probably want to add a couple more cases with edge cases like total = 0 GB, free = 0 Gb.